README.md

```
@@ -10,3 +10,16 @@ PHPUnitテスト
 npm run phpunit
 ```
 
+Playwright e2e テスト
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)